content/pt-br/docs/reference/setup-tools/kubeadm/generated/kubeadm_upgrade_node_phase_kubelet-config.md

```diff
@@ -0,0 +1,77 @@
+<!--
+The file is auto-generated from the Go source code of the component using a generic
+[generator](https://github.com/kubernetes-sigs/reference-docs/). To learn how
+to generate the reference documentation, please read
+[Contributing to the reference documentation](/docs/contribute/generate-ref-docs/).
+To update the reference conent, please follow the 
+[Contributing upstream](/docs/contribute/generate-ref-docs/contribute-upstream/)
+guide. You can file document formatting bugs against the
+[reference-docs](https://github.com/kubernetes-sigs/reference-docs/) project.
+-->
+
+
+Atualize a configuração do kubelet para este nó
+
+### Sinopse
+
+No cluster baixe a configuração do kubelet do ConfigMap no formato "kubelet-config-1.X", onde X é a menor versão do kubelet. O kubeadm usa o campo KuberneteVersion no kubeadm-config ConfigMap para determinar qual é a versão _desejada_ do kubelet.
```

Comment:
Suggestions to improve readability:
* Maybe instead of "No cluster baixe", "Baixa no cluster"?
* Instead of "a configuração do kubelet do ConfigMap", maybe "o ConfigMap de configuração do kubelet"?
* Instead of "kubeadm-config ConfigMap", "ConfigMap kubeadm-config"?